.github/workflows/on_prerelease.yaml

```diff
@@ -9,8 +9,10 @@ on:
 
 jobs:
   pre-release:
-    uses: newrelic/coreint-automation/.github/workflows/reusable_pre_release.yaml@v3
+    uses: newrelic/coreint-automation/.github/workflows/reusable_pre_release.yaml@test-fips-s3-upload-schema
```

Comment:
I understand this was only used while development and you'll update the v3 once merged, isn't it?

Reply:
Yes, correct. I'll change it after https://github.com/newrelic/coreint-automation/pull/93 is merged.

Reply:
done